opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "indexManagementDashboards",
-  "version": "2.0.0.0-rc1",
```

Comment:
Does this mean that this isn't going into the preview? or have we been told to change this?

Reply:
Oh this might have been from my commit.. I remember Dashboards wasn't starting up with this in here so I had changed it while testing. If that's not still an issue we should revert.

Reply:
updated